pages/common/volta.md

```diff
@@ -0,0 +1,28 @@
+# volta
+
+> A JavaScript Tool Manager that installs Node.js runtimes, Npm and Yarn package managers, or any binaries from Npm.
+> More information: <https://volta.sh>.
+
+- List all installed tools:
+
+`volta list`
+
+- Install the latest version of a tool:
+
+`volta install {{node|npm|yarn}}`
```

Comment:
This is a shortcut since we can also install any binaries. Not sure if this if this example is better:

> `volta install {{node|npm|yarn|any-binary-from-npm}}`

Reply:
Or something like this:

```suggestion
`volta install {{npm_package|node}}`
```

Considering that both `npm` and `yarn` are technically NPM packages, too (+	1 for recursion), I think this can be simplified this way.

Reply:
Not sure: your proposal is technically better but I think many users just want to install `npm`, `node` or `yarn`. 🤔

Reply:
`node|npm|yarn|npm_package_name`?

Reply:
```suggestion
`volta install {{node|npm|yarn|package_name}}`
```